lib/Mixpanel.php

```diff
@@ -149,10 +149,10 @@ public function __construct($token, $options = array()) {
      * @return Mixpanel
      */
     public static function getInstance($token, $options = array()) {
-        if(!isset(self::$_instance)) {
-            self::$_instance = new Mixpanel($token, $options);
+        if(!isset(self::$_instances[$token])) {
```

Comment:
Nope, there will be no notice. `isset($a['b']) == array_key_exists('b', $a) && !is_null($a['b'])`